SOP/SOP-push.adoc

```diff
@@ -185,43 +182,44 @@ Init Containers:
     State:          Terminated
       Reason:       Completed
       Exit Code:    0
-      Started:      Fri, 12 Jul 2019 08:51:28 -0300
-      Finished:     Fri, 12 Jul 2019 08:51:53 -0300
+      Started:      Thu, 26 Sep 2019 12:09:52 +0300
+      Finished:     Thu, 26 Sep 2019 12:09:52 +0300
     Ready:          True
     Restart Count:  0
     Environment:
       POSTGRES_SERVICE_HOST:  example-unifiedpushserver-postgresql
     Mounts:
-      /var/run/secrets/kubernetes.io/serviceaccount from example-unifiedpushserver-token-5cmlp (ro)
+      /var/run/secrets/kubernetes.io/serviceaccount from example-unifiedpushserver-token-cszr4 (ro)
 Containers:
   ups:
-    Container ID:   docker://28fc71000c66d9223ab6e340a030491c4348a0b51979237de04488fe18282337
-    Image:          docker.io/aerogear/unifiedpush-wildfly-plain@sha256:62ecab1e74e3b1a7b2ef1d9eb7594f29bcf6b55702c269c9deebdadf8aea8a8a
-    Image ID:       docker-pullable://docker.io/aerogear/unifiedpush-wildfly-plain@sha256:62ecab1e74e3b1a7b2ef1d9eb7594f29bcf6b55702c269c9deebdadf8aea8a8a
+    Container ID:   docker://2cdc9c3703f274053e724b589ac24e1ba12db52a07158b68ffd7ddec328fcb51
+    Image:          quay.io/aerogear/unifiedpush-configurable-container:2.3
```

Comment:
I had 2.4

Reply:
same as above. rebased and then forgot this.